### PR TITLE
Allow alternative names for bootstrap packages

### PR DESCRIPTION
**Pull request by juliogonzalez**
## What does this PR change?

Allow alternative names for bootstrap packages, to allow using old client tools after package renames

SLE12-SP0, released client tools (no mgr-daemon available):

Script uses `spacewalksd` as `mgr-daemon` is not available. And if a package has alternatives and none is found, the error will be more explicit than @mcalmer suggestion (for packages with alternatives the error is the same it was).

Note the error with python-setup being missing is a bug to be fixed separately, so I added an alternative `python3-setuptools` to should how the failure when there are alternatives.

```
suma40-server:/var/log/rhn/reposync # mgr-create-bootstrap-repo -f    
1. SLE-12-SP4-x86_64
2. SLE-12-x86_64
Enter a number of a product label: 2
Removing directory: /srv/www/htdocs/pub/repositories/sle/12/0/bootstrap/
Creating bootstrap repo for SLE-12-x86_64

copy 'dbus-1-python-1.2.0-4.194.x86_64'
copy 'hwdata-0.314-10.9.1.noarch'
copy 'libcurl4-7.37.0-21.1.x86_64'
copy 'libgudev-1_0-0-210-70.51.1.x86_64'
copy 'libnewt0_52-0.52.16-1.83.x86_64'
copy 'libnl1-1.1.4-4.21.x86_64'
copy 'libopenssl1_0_0-1.0.1i-27.16.1.x86_64'
copy 'libsqlite3-0-3.8.3.1-2.3.1.x86_64'
copy 'libudev1-210-70.51.1.x86_64'
copy 'udev-210-70.51.1.x86_64'
copy 'openssl-1.0.1i-27.16.1.x86_64'
copy 'python-libxml2-2.9.1-24.1.x86_64'
copy 'libzypp-14.43.0-2.43.1.x86_64'
copy 'newt-0.52.16-1.83.x86_64'
copy 'python-2.7.9-14.1.x86_64'
copy 'python-cffi-0.8.2-1.10.x86_64'
copy 'python-cryptography-0.4-1.23.x86_64'
copy 'python-dmidecode-3.12.1-13.1.x86_64'
copy 'python-ethtool-0.9-3.2.x86_64'
copy 'python-gobject2-2.28.6-24.80.x86_64'
copy 'python-gudev-147.2-3.1.x86_64'
copy 'python2-hwdata-2.3.5-12.3.1.noarch'
copy 'python-newt-0.52.16-1.83.x86_64'
copy 'python-pyasn1-0.1.7-5.7.noarch'
copy 'python-pycparser-2.10-3.7.noarch'
copy 'python-pyOpenSSL-0.14-1.7.noarch'
copy 'python-six-1.9.0-9.7.1.noarch'
copy 'python-xml-2.7.9-14.1.x86_64'
copy 'python-pyudev-0.16.1-1.11.noarch'
copy 'python2-rhnlib-2.8.11.1-21.6.1.noarch'
copy 'rpm-python-4.11.2-10.1.x86_64'
copy 'libsolv-tools-0.6.20-2.20.1.x86_64'
copy 'libslang2-2.2.4-9.7.x86_64'
copy 'spacewalk-check-2.8.22.4-52.12.1.noarch'
copy 'python2-spacewalk-check-2.8.22.4-52.12.1.noarch'
copy 'spacewalk-client-setup-2.8.22.4-52.12.1.noarch'
copy 'python2-spacewalk-client-setup-2.8.22.4-52.12.1.noarch'
copy 'spacewalk-client-tools-2.8.22.4-52.12.1.noarch'
copy 'python2-spacewalk-client-tools-2.8.22.4-52.12.1.noarch'
copy 'spacewalk-usix-2.8.3.1-3.3.1.noarch'
copy 'spacewalksd-5.0.37.1-24.9.1.x86_64'
copy 'suseRegisterInfo-3.1.1-24.2.x86_64'
copy 'suseRegisterInfo-3.2.2-25.3.1.noarch'
copy 'python2-suseRegisterInfo-3.2.2-25.3.1.noarch'
copy 'zypp-plugin-python-0.5-2.3.1.x86_64'
copy 'zypp-plugin-spacewalk-0.9.16-30.3.1.x86_64'
copy 'zypp-plugin-spacewalk-1.0.4-30.12.1.noarch'
copy 'python2-zypp-plugin-spacewalk-1.0.4-30.12.1.noarch'
copy 'zypper-1.11.54-2.38.3.x86_64'
copy 'yast2-packager-3.1.52.1-7.3.3.x86_64'
copy 'yast2-pkg-bindings-3.1.20.3-11.1.x86_64'
copy 'python-backports.ssl_match_hostname-3.5.0.1-21.3.1.noarch'
copy 'python-futures-3.0.2-7.1.noarch'
copy 'python-Jinja2-2.8-19.14.1.noarch'
copy 'python-MarkupSafe-0.18-16.3.1.x86_64'
copy 'python-msgpack-python-0.4.6-8.3.1.x86_64'
copy 'python-psutil-5.2.2-15.6.2.x86_64'
copy 'python-pycrypto-2.6.1-10.6.3.x86_64'
copy 'python-PyYAML-3.12-26.3.1.x86_64'
copy 'python-pyzmq-14.0.0-9.3.1.x86_64'
copy 'python-requests-2.11.1-6.25.1.noarch'
copy 'python-simplejson-3.3.1-1.45.x86_64'
copy 'python-tornado-4.2.1-17.3.1.x86_64'
copy 'salt-2018.3.0-46.59.1.x86_64'
copy 'salt-minion-2018.3.0-46.59.1.x86_64'
copy 'python2-salt-2018.3.0-46.59.1.x86_64'
copy 'libgio-2_0-0-2.38.2-5.12.x86_64'
copy 'libgthread-2_0-0-2.38.2-5.12.x86_64'
copy 'shared-mime-info-1.2-7.1.x86_64'
copy 'glib2-tools-2.38.2-5.12.x86_64'
copy 'libelf0-0.8.13-18.64.x86_64'
copy 'logrotate-3.8.7-3.21.x86_64'
copy 'cron-4.2-58.3.x86_64'
copy 'cronie-1.4.11-58.3.x86_64'
copy 'libzmq3-4.0.4-6.1.x86_64'
copy 'gio-branding-SLE-12-6.9.noarch'
copy 'wallpaper-branding-SLE-12-10.41.noarch'
copy 'libyui-ncurses-pkg6-2.46.1-3.4.x86_64'
copy 'libyui-qt-pkg6-2.44.7-3.2.x86_64'
copy 'PackageKit-backend-zypp-0.8.16-13.3.10.x86_64'
copy 'PackageKit-lang-0.8.16-13.3.10.noarch'
Directory walk started
Directory walk done - 81 packages
Temporary output repo path: /srv/www/htdocs/pub/repositories/sle/12/0/bootstrap/.repodata/
Preparing sqlite DBs
Pool started (with 5 workers)
Pool finished
ERROR: none of 'python-setuptools', 'python3-setuptools' found

Suggestions:
- mgr-create-bootstrap-repo uses the locally synchronized versions of files
  from the Tools repository, and uses the locally synchronized pool channel
  for dependency resolution. Both should be fully synced before running the
  mgr-create-bootstrap-repo script.
```

SLE12 SP4, released client tools (no mgr-daemon available):

Script uses `spacewalksd` as `mgr-daemon` is not available. 
```
suma40-server:/var/log/rhn/reposync # mgr-create-bootstrap-repo -f 
1. SLE-12-SP4-x86_64
2. SLE-12-x86_64
Enter a number of a product label: 1
Removing directory: /srv/www/htdocs/pub/repositories/sle/12/4/bootstrap/
Creating bootstrap repo for SLE-12-SP4-x86_64

copy 'dbus-1-python-1.2.0-4.194.x86_64'
copy 'hwdata-0.314-10.9.1.noarch'
copy 'libcurl4-7.60.0-4.3.1.x86_64'
copy 'libgudev-1_0-0-230-1.5.x86_64'
copy 'libnewt0_52-0.52.16-1.83.x86_64'
copy 'libnl1-1.1.4-4.21.x86_64'
copy 'libopenssl1_0_0-1.0.2p-3.6.1.x86_64'
copy 'libsqlite3-0-3.8.10.2-8.1.x86_64'
copy 'libudev1-228-150.63.1.x86_64'
copy 'udev-228-150.63.1.x86_64'
copy 'openssl-1.0.2p-1.13.noarch'
copy 'python-libxml2-2.9.4-46.15.1.x86_64'
copy 'libzypp-16.19.0-2.36.3.x86_64'
copy 'newt-0.52.16-1.83.x86_64'
copy 'python-2.7.13-28.21.1.x86_64'
copy 'python-cffi-1.5.2-5.6.2.x86_64'
copy 'python-cryptography-1.3.1-7.13.4.x86_64'
copy 'python-dmidecode-3.12.1-13.1.x86_64'
copy 'python-ethtool-0.11-5.2.x86_64'
copy 'python-gobject2-2.28.6-24.80.x86_64'
copy 'python-gudev-147.2-3.1.x86_64'
copy 'python2-hwdata-2.3.5-12.3.1.noarch'
copy 'python-newt-0.52.16-1.83.x86_64'
copy 'python-pyasn1-0.1.9-4.6.1.noarch'
copy 'python-pycparser-2.10-5.6.1.noarch'
copy 'python-pyOpenSSL-16.0.0-4.17.1.noarch'
copy 'python-setuptools-18.0.1-4.8.1.noarch'
copy 'python3-setuptools-18.0.1-4.8.1.noarch'
copy 'python-six-1.11.0-9.18.1.noarch'
copy 'python-xml-2.7.13-28.21.1.x86_64'
copy 'python-pyudev-0.16.1-3.2.1.noarch'
copy 'python2-rhnlib-2.8.11.1-21.6.1.noarch'
copy 'rpm-python-4.11.2-16.21.1.x86_64'
copy 'libsolv-tools-0.6.34-2.13.1.x86_64'
copy 'libslang2-2.2.4-9.7.x86_64'
copy 'spacewalk-check-2.8.22.4-52.12.1.noarch'
copy 'python2-spacewalk-check-2.8.22.4-52.12.1.noarch'
copy 'spacewalk-client-setup-2.8.22.4-52.12.1.noarch'
copy 'python2-spacewalk-client-setup-2.8.22.4-52.12.1.noarch'
copy 'spacewalk-client-tools-2.8.22.4-52.12.1.noarch'
copy 'python2-spacewalk-client-tools-2.8.22.4-52.12.1.noarch'
copy 'spacewalk-usix-2.8.3.1-3.3.1.noarch'
copy 'spacewalksd-5.0.37.1-24.9.1.x86_64'
copy 'suseRegisterInfo-3.1.1-24.2.x86_64'
copy 'suseRegisterInfo-3.2.2-25.3.1.noarch'
copy 'python2-suseRegisterInfo-3.2.2-25.3.1.noarch'
copy 'zypp-plugin-python-0.6.3-3.6.1.noarch'
copy 'zypp-plugin-spacewalk-0.9.16-30.3.1.x86_64'
copy 'zypp-plugin-spacewalk-1.0.4-30.12.1.noarch'
copy 'python2-zypp-plugin-spacewalk-1.0.4-30.12.1.noarch'
copy 'zypper-1.13.45-21.23.4.x86_64'
copy 'yast2-packager-3.2.30-2.13.12.x86_64'
copy 'yast2-pkg-bindings-3.2.5-2.6.36.x86_64'
copy 'python-backports.ssl_match_hostname-3.5.0.1-21.3.1.noarch'
copy 'python-futures-3.0.2-7.1.noarch'
copy 'python-Jinja2-2.8-19.14.1.noarch'
copy 'python-MarkupSafe-0.18-16.3.1.x86_64'
copy 'python-msgpack-python-0.4.6-8.3.1.x86_64'
copy 'python-psutil-5.2.2-15.6.2.x86_64'
copy 'python-pycrypto-2.6.1-10.6.3.x86_64'
copy 'python-PyYAML-3.12-26.6.1.x86_64'
copy 'python-pyzmq-14.0.0-9.3.1.x86_64'
copy 'python-requests-2.11.1-6.28.1.noarch'
copy 'python-simplejson-3.8.2-12.1.x86_64'
copy 'python-tornado-4.2.1-17.3.1.x86_64'
copy 'salt-2018.3.0-46.59.1.x86_64'
copy 'salt-minion-2018.3.0-46.59.1.x86_64'
copy 'python2-salt-2018.3.0-46.59.1.x86_64'
copy 'libgio-2_0-0-2.48.2-10.2.x86_64'
copy 'libgthread-2_0-0-2.48.2-10.2.x86_64'
copy 'shared-mime-info-1.6-11.3.x86_64'
copy 'glib2-tools-2.48.2-10.2.x86_64'
copy 'libelf0-0.8.13-18.64.x86_64'
copy 'logrotate-3.11.0-2.11.1.x86_64'
copy 'cron-4.2-59.3.2.x86_64'
copy 'cronie-1.4.11-59.3.2.x86_64'
copy 'libzmq3-4.0.4-6.1.x86_64'
copy 'gio-branding-SLE-12-14.5.noarch'
copy 'wallpaper-branding-SLE-12-13.3.1.noarch'
copy 'libyui-ncurses-pkg7-2.48.4.1-1.5.x86_64'
copy 'libyui-qt-pkg7-2.45.13.1-1.6.x86_64'
copy 'python-enum34-1.1.3-7.1.noarch'
copy 'python-idna-2.0-3.6.1.noarch'
copy 'python-ipaddress-1.0.18-3.9.1.noarch'
copy 'python-setuptools-18.0.1-4.8.1.noarch'
Directory walk started
Directory walk done - 84 packages
Temporary output repo path: /srv/www/htdocs/pub/repositories/sle/12/4/bootstrap/.repodata/
Preparing sqlite DBs
Pool started (with 5 workers)
Pool finished
```

SLE12 SP4, with beta client tools available as a custom channel (mgr-daemon available):

Script uses `mgr-daemon` as it is available. 
```
suma40-server:/var/log/rhn/reposync # mgr-create-bootstrap-repo -f --with-custom-channels
1. SLE-12-SP4-x86_64
2. SLE-12-x86_64
Enter a number of a product label: 1
Removing directory: /srv/www/htdocs/pub/repositories/sle/12/4/bootstrap/
Creating bootstrap repo for SLE-12-SP4-x86_64

copy 'dbus-1-python-1.2.0-4.194.x86_64'
copy 'hwdata-0.314-10.12.1.noarch'
copy 'libcurl4-7.60.0-4.3.1.x86_64'
copy 'libgudev-1_0-0-230-1.5.x86_64'
copy 'libnewt0_52-0.52.16-1.83.x86_64'
copy 'libnewt0_52-0.52.16-1.83.x86_64'
copy 'libnl1-1.1.4-4.21.x86_64'
copy 'libopenssl1_0_0-1.0.2p-3.6.1.x86_64'
copy 'libsqlite3-0-3.8.10.2-8.1.x86_64'
copy 'libudev1-228-150.63.1.x86_64'
copy 'udev-228-150.63.1.x86_64'
copy 'openssl-1.0.2p-1.13.noarch'
copy 'python-libxml2-2.9.4-46.15.1.x86_64'
copy 'libzypp-16.19.0-2.36.3.x86_64'
copy 'newt-0.52.16-1.83.x86_64'
copy 'newt-0.52.16-1.83.x86_64'
copy 'python-2.7.13-28.21.1.x86_64'
copy 'python-cffi-1.5.2-5.6.2.x86_64'
copy 'python-cryptography-1.3.1-7.13.4.x86_64'
copy 'python-dmidecode-3.12.1-13.1.x86_64'
copy 'python-dmidecode-3.12.1-13.1.x86_64'
copy 'python-ethtool-0.11-5.2.x86_64'
copy 'python-gobject2-2.28.6-24.80.x86_64'
copy 'python-gudev-147.2-6.1.x86_64'
copy 'python2-hwdata-2.3.5-12.3.1.noarch'
copy 'python2-hwdata-2.3.5-12.3.1.noarch'
copy 'python-newt-0.52.16-1.83.x86_64'
copy 'python-newt-0.52.16-1.83.x86_64'
copy 'python-pyasn1-0.1.9-4.6.1.noarch'
copy 'python-pycparser-2.10-5.6.1.noarch'
copy 'python-pyOpenSSL-16.0.0-4.17.1.noarch'
copy 'python-setuptools-18.0.1-4.8.1.noarch'
copy 'python3-setuptools-18.0.1-4.8.1.noarch'
copy 'python-six-1.11.0-9.18.1.noarch'
copy 'python-xml-2.7.13-28.21.1.x86_64'
copy 'python-pyudev-0.16.1-3.2.1.noarch'
copy 'python2-rhnlib-4.0.5-21.12.1.noarch'
copy 'rpm-python-4.11.2-16.21.1.x86_64'
copy 'libsolv-tools-0.6.34-2.13.1.x86_64'
copy 'libslang2-2.2.4-9.7.x86_64'
copy 'spacewalk-check-4.0.6-52.14.1.noarch'
copy 'python2-spacewalk-check-4.0.6-52.14.1.noarch'
copy 'spacewalk-client-setup-4.0.6-52.14.1.noarch'
copy 'python2-spacewalk-client-setup-4.0.6-52.14.1.noarch'
copy 'spacewalk-client-tools-4.0.6-52.14.1.noarch'
copy 'python2-spacewalk-client-tools-4.0.6-52.14.1.noarch'
copy 'spacewalk-usix-4.0.5-3.8.1.noarch'
copy 'mgr-daemon-4.0.3-1.4.noarch'
copy 'spacewalksd-5.0.37.1-24.9.1.x86_64'
copy 'suseRegisterInfo-3.1.1-24.2.x86_64'
copy 'suseRegisterInfo-4.0.2-25.6.3.noarch'
copy 'python2-suseRegisterInfo-4.0.2-25.6.3.noarch'
copy 'zypp-plugin-python-0.6.3-3.6.1.noarch'
copy 'zypp-plugin-spacewalk-0.9.16-30.3.1.x86_64'
copy 'zypp-plugin-spacewalk-1.0.4-30.12.1.noarch'
copy 'zypp-plugin-spacewalk-0.9.16-30.3.1.x86_64'
copy 'python2-zypp-plugin-spacewalk-1.0.4-30.12.1.noarch'
copy 'python2-zypp-plugin-spacewalk-1.0.4-30.12.1.noarch'
copy 'zypper-1.13.45-21.23.4.x86_64'
copy 'yast2-packager-3.2.30-2.13.12.x86_64'
copy 'yast2-pkg-bindings-3.2.5-2.6.36.x86_64'
copy 'python-backports.ssl_match_hostname-3.5.0.1-21.3.1.noarch'
copy 'python-futures-3.0.2-7.1.noarch'
copy 'python-Jinja2-2.8-19.14.1.noarch'
copy 'python-MarkupSafe-0.18-16.3.1.x86_64'
copy 'python-msgpack-python-0.4.6-8.3.1.x86_64'
copy 'python-psutil-5.2.2-15.6.2.x86_64'
copy 'python-pycrypto-2.6.1-10.6.3.x86_64'
copy 'python-PyYAML-3.12-26.6.1.x86_64'
copy 'python-pyzmq-14.0.0-9.3.1.x86_64'
copy 'python-requests-2.11.1-6.28.1.noarch'
copy 'python-simplejson-3.8.2-12.1.x86_64'
copy 'python-tornado-4.2.1-17.3.1.x86_64'
copy 'salt-2019.2.0-46.54.1.x86_64'
copy 'salt-minion-2019.2.0-46.54.1.x86_64'
copy 'python2-salt-2019.2.0-46.54.1.x86_64'
copy 'libgio-2_0-0-2.48.2-10.2.x86_64'
copy 'libgthread-2_0-0-2.48.2-10.2.x86_64'
copy 'shared-mime-info-1.6-11.3.x86_64'
copy 'glib2-tools-2.48.2-10.2.x86_64'
copy 'libelf0-0.8.13-18.64.x86_64'
copy 'logrotate-3.11.0-2.11.1.x86_64'
copy 'cron-4.2-59.3.2.x86_64'
copy 'cronie-1.4.11-59.3.2.x86_64'
copy 'libzmq3-4.0.4-14.1.x86_64'
copy 'gio-branding-SLE-12-14.5.noarch'
copy 'wallpaper-branding-SLE-12-13.3.1.noarch'
copy 'libyui-ncurses-pkg7-2.48.4.1-1.5.x86_64'
copy 'libyui-qt-pkg7-2.45.13.1-1.6.x86_64'
copy 'python-enum34-1.1.3-7.1.noarch'
copy 'python-idna-2.0-3.6.1.noarch'
copy 'python-ipaddress-1.0.18-3.9.1.noarch'
copy 'python-setuptools-18.0.1-4.8.1.noarch'
Directory walk started
Directory walk done - 85 packages
Temporary output repo path: /srv/www/htdocs/pub/repositories/sle/12/4/bootstrap/.repodata/
Preparing sqlite DBs
Pool started (with 5 workers)
Pool finished
```
## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed: The list of packages is not to be edited by users.

- [x] **DONE**

## Test coverage
- No tests: Cucumber tests already test this. And as far as I can see, we don't have tests.

- [x] **DONE**

## Links

Fixes  https://github.com/SUSE/spacewalk/issues/7219
- [x] **DONE**

### CODE REVIEW

File: susemanager/src/mgr-create-bootstrap-repo [hunk 1]
```diff
@@ -247,28 +247,29 @@ def create_repo(label, flush, additional=[]):
...
-        for p in pkgs:
-            logging.debug(p)
-            rpmdir = os.path.join(destdir, p['arch'])
-            if not os.path.exists(rpmdir):
-                os.makedirs(rpmdir)
-            print("copy '%s'" % p['nvrea'])
-            logging.debug("copy {0} / {1} to {2}".format(basepath, p['path'], rpmdir))
-            if not dryrun:
-                shutil.copy2(os.path.join(basepath, p['path']), rpmdir)
+        for pkgname in pkgname.split("|"):
```
Comment (juliogonzalez) [inline susemanager/src/mgr-create-bootstrap-repo]: This works (the same variable can be reused).

We could consider, however, doing the split before to have a separate variable...

File: susemanager/src/mgr-create-bootstrap-repo [hunk 2]
```diff
@@ -247,28 +247,29 @@ def create_repo(label, flush, additional=[]):
...
+            h.execute(parentchannel=options.parentchannel, pkgname=pkgname)
+            pkgs = h.fetchall_dict() or []
+            logging.debug("Package {0} found {1} resulting packages:".format(
+                pkgname, len(pkgs)))
+            if len(pkgs) == 0:
+                messages.append("ERROR: package '%s' not found" % pkgname)
```
Comment (juliogonzalez) [inline susemanager/src/mgr-create-bootstrap-repo]: ... so we can use it here:
```
if len(pkglistname) > 0:
    messages.append("INFO: package '%s' has an alternative, trying to locate it now" %pkgname)
```

**Comment by juliogonzalez** [comment]
This is a crude solution, IMHO (if there are alternatives maybe we could just NOT show an error to the user unless no alternatives are found), but I am not sure if we should invest much more time of this, as package renames are not common.

Opinions, please.

**Comment by mcalmer** [comment]
```
@@ -246,20 +246,24 @@ def create_repo(label, flush, additional=[]):
     packagelist.extend(additional)
     logging.debug("The bootstrap repo should contain the following packages: {0}".format(packagelist))
 
-    for pkgname in packagelist:
-        for pkgname in pkgname.split("|"):
+    for pkgaltstr in packagelist:
+        alt = pkgaltstr.split("|")
+        altcount = 0
+        for pkgname in alt:
+            altcount += 1
             h.execute(parentchannel=options.parentchannel, pkgname=pkgname)
             pkgs = h.fetchall_dict() or []
             logging.debug("Package {0} found {1} resulting packages:".format(
                 pkgname, len(pkgs)))
             if len(pkgs) == 0:
-                messages.append("ERROR: package '%s' not found" % pkgname)
-                errors += 1
-                if not suggestions['no-packages']:
-                    suggestions['no-packages'] = ('mgr-create-bootstrap-repo uses the locally synchronized ' +
-                                                  'versions of files from the Tools repository, ' +
-                                                  'and uses the locally synchronized pool channel for dependency resolution. ' +
-                                                  'Both should be fully synced before running the mgr-create-bootstrap-repo script.')
+                if altcount >= len(alt):
+                    messages.append("ERROR: package '%s' not found" % pkgname)
+                    errors += 1
+                    if not suggestions['no-packages']:
+                        suggestions['no-packages'] = ('mgr-create-bootstrap-repo uses the locally synchronized ' +
+                                                      'versions of files from the Tools repository, ' +
+                                                      'and uses the locally synchronized pool channel for dependency resolution. ' +
+                                                      'Both should be fully synced before running the mgr-create-bootstrap-repo script.')
                 continue
             for p in pkgs:
                 logging.debug(p)
```

I do not like to overwrite the variable and printing errors if there is an alternative is also not good.
Here is a suggested patch

**Comment by juliogonzalez** [comment]
Yes, you basically converted my crude patch into the more elaborate alternative I had in mind.

However this is still not telling the user about alternatives, and if I understand correct, will show that the script is looking for two packages, but will only produce an error in case the last package in the alternatives is not found.

I think we could still improve this a little bit more by adding an `INFO:` with the packages, provided `len(alt) > 1`, and then when we provide the error state that neither packages were found.

If you agree, I will implement it and test it tomorrow.

**Comment by juliogonzalez** [comment]
Waiting for the bot and merging.

**Comment by mcalmer** [comment]
gitarro rerun changelog_test !!!

**Comment by mcalmer** [comment]
@juliogonzalez as we rename the package in the clients tools it will also affect 3.2.
I think we should backport it to 3.2

**Comment by juliogonzalez** [comment]
@mcalmer, ACK. Creating backport now.

**Pull request merged**